test/Locker.t.sol

```diff
@@ -8,38 +8,14 @@ contract LockerTest is Test {
     address constant ADDRESS_AS = 0xaAaAaAaaAaAaAaaAaAAAAAAAAaaaAaAaAaaAaaAa;
     address constant ADDRESS_BS = 0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB;
 
-    function test_fuzz_setLocker(address locker) public {
-        assertEq(address(Locker.getLocker()), address(0));
-
-        if (locker == address(0)) {
-            vm.expectRevert(Locker.InvalidLocker.selector);
-            Locker.setLocker(locker);
-        } else {
-            Locker.setLocker(locker);
-            assertEq(address(Locker.getLocker()), locker);
-        }
-    }
-
-    function test_fuzz_clearLocker(address locker) public {
-        vm.assume(locker != address(0));
-        Locker.setLocker(locker);
-
-        assertEq(address(Locker.getLocker()), locker);
-
-        Locker.clearLocker();
-
-        assertEq(address(Locker.getLocker()), address(0));
-    }
-
-    function test_fuzz_isLocked(address locker) public {
-        vm.assume(locker != address(0));
+    function test_fuzz_lock() public {
```

Comment:
this is no longer a fuzz test - fuzz tests are when the test has a parameter like `address locker` and forge will choose 100,000 different values for that parameter and check that the test passes for all of them. Now that we dont have a parameter this can just be `test_lock`